pkg/apis/admissionregistration/v1alpha1/doc.go

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 
 // Package v1alpha1 is the v1alpha1 version of the API.
 // AdmissionConfiguration and AdmissionPluginConfiguration are legacy static admission plugin configuration
-// admissionregistrationv1alpha1.InitializerConfiguration and admissionregistrationv1alpha1.ExternalAdmissionHookConfiguration is for the
+// admissionregistrationv1alpha1.InitializerConfiguration and admissionregistrationv1alpha1.ValidatingWebhookConfiguration is for the
 // new dynamic admission controller configuration.
 // +groupName=admissionregistration.k8s.io
```

Comment:
You might want `// +groupGoName=AdmissionRegistration` here to get prettier funcs in the clientsets.

Reply:
Sure. Do we have an issue to collect TODOs for webhook admission?